project/assets/test/turbofat-24cc.json

```diff
@@ -101,5 +101,258 @@
+        "#filler-105#": 1,
+        "#filler-106#": 1.173456,
+        "#filler-107#": 1,
+        "#filler-108#": 1,
+        "#filler-109#": 1.603162,
+        "#filler-110#": 1.875,
+        "#filler-111#": 1.212497,
+        "#filler-112#": 1.065797,
+        "#filler-113#": 1.504414,
+        "#filler-114#": 1,
+        "#filler-115#": 1.777527,
+        "#filler-116#": 2.5,
+        "#filler-117#": 2.118348,
+        "#filler-118#": 1.502927,
+        "#filler-119#": 1,
+        "#filler-120#": 1,
+        "#filler-121#": 2.179599,
+        "#filler-122#": 1.439336,
+        "#filler-123#": 1.555465,
+        "#filler-124#": 1,
+        "#filler-125#": 2.5,
+        "#filler-126#": 1.671844,
+        "#filler-127#": 1,
+        "#filler-128#": 1,
+        "#filler-129#": 1.341536,
+        "#filler-130#": 1,
+        "#filler-131#": 1.019266,
+        "#filler-132#": 2.144976,
+        "#filler-133#": 2.013286,
+        "#filler-134#": 1,
+        "#filler-135#": 1.609133,
+        "#filler-136#": 2.5,
+        "#filler-137#": 1.06066,
+        "#filler-138#": 1.583015,
+        "#filler-139#": 1.064168,
+        "#filler-140#": 2.320883,
+        "#filler-141#": 1.344991,
+        "#filler-142#": 2.5,
+        "#filler-143#": 1.909188,
+        "#filler-144#": 1,
+        "#filler-145#": 1,
+        "#filler-146#": 1.904764,
+        "#filler-147#": 2.199352,
+        "#filler-148#": 1.573205,
+        "#filler-149#": 1,
+        "dingold": 1.2,
+        "snorz": 1.805482,
+        "rhinosaur": 1.647459,
+        "chelle": 2.124839,
+        "squawkapus": 2.074417,
+        "gongo": 1.644812,
+        "redo": 2.421101,
+        "rice": 1,
+        "ricket": 1.2,
+        "nictor": 2.991535,
+        "gall": 1,
+        "dreadhark": 1.173456,
+        "leentil": 2.711691,
+        "sidger": 4.301399,
+        "bundles": 2.53754,
+        "flutter": 2.546443,
+        "beats": 1.005112,
+        "nardine": 1.2,
+        "phthalo": 1.917655,
+        "sergia": 1,
+        "tork": 1,
+        "goris": 2.526987,
+        "nard": 1.717901,
+        "mocha": 3.639716,
+        "alber": 1.1,
+        "miteby": 1.270802,
+        "raymon": 1,
+        "rex": 1.026158,
+        "eeeroy": 1.946466,
+        "lango": 1.653406,
+        "darry": 1.705009,
+        "eech": 2.2199,
+        "lisk": 1,
+        "spood": 1.4,
+        "kurtles": 1.194856,
+        "diota": 1.1,
+        "jayton": 1.286897,
+        "alexander": 2.623928,
+        "minimax": 1.011805,
+        "goldfince": 1.740517,
+        "pandon": 1.448413,
+        "pipedro": 2.479496,
+        "ranpo": 2.82319,
+        "cast": 1.505988,
+        "stunker": 1.3,
+        "logana": 1.930285,
+        "magpird": 1.2,
+        "vile": 1.1,
+        "rhonk": 1,
+        "mewow": 1,
+        "pairet": 1.1,
+        "tunathy": 1.69334,
+        "gordo": 1,
+        "terpion": 1.207477,
+        "sandrew": 1.887856,
+        "i-n-cognito": 1.3
+      }
+    }
```

Comment:
This section should be in the new turbofat-2743 save file, and begs the question of how on earth the current tests passed

Reply:
Moved to `turbofat-2743.save`. The tests were passing because CreatureLibrary.reset() did not erase fatness data, so the data was carried forward from the 24cc test. I've fixed CreatureLibrary.reset().